src/txmempool.h

```diff
@@ -821,7 +818,15 @@ class CTxMemPool
     }
     bool DumpMempoolPath(const fs::path& dump_path, FopenFn mockable_fopen_function = fsbridge::fopen, bool skip_file_commit = false) const;
 
+    /** Try to load the mempool from disk. */
+    void TryLoadMempool(std::function<bool(const CTransactionRef&, int64_t)> simple_atmp, FopenFn mockable_fopen_function = fsbridge::fopen);
+
 private:
+    /** Sets the current loaded state */
+    void SetLoadTried(bool loaded);
+
+    bool LoadMempool(std::function<bool(const CTransactionRef&, int64_t)> simple_atmp, FopenFn mockable_fopen_function = fsbridge::fopen);
```

Comment:
can get a comment

Reply:
The newest version of this PR doesn't have `simple_atmp`, and the new `::LoadMempool` signature seem simple enough, feel free to reopen if still a problem!